File: Assignment5/C/Program3.c
// PS - Write a program which accepts number from user anf print numbers line
// EX -> i/p - 5
// o/p -> -5  -4  -3  -2  -1  0   1   2   3   4   5

#include<stdio.h>

void PrintNum(int iNo) {
    int iCnt = 0;

    for(iCnt = -iNo; iCnt <= iNo; iCnt++) {
        printf("%d \t", iCnt);
    }
}

int main() {
    int iValue = 0;

    printf("Enter number \n");
    scanf("%d", &iValue);

    PrintNum(iValue);

    return 0;
}

File: Assignment34/Java/Program3.java
// PS - Write a java program which accept N numbers from user and display all such numbers which are divisible by 5 and 3

import java.util.*;

class ArrayX
{
    public void DivisibleBy5and3(int Arr[])
    {
        for(int iCnt = 0; iCnt< Arr.length; iCnt++)
        {
            if(((Arr[iCnt] % 5) == 0) && ((Arr[iCnt] % 3) == 0))
            {
                System.out.print(Arr[iCnt]+"\t");
            }
        }
    }
}

class Program3
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iSize = 0;
        System.out.println("Enter number of elements ");
        iSize = sobj.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter elements ");
        for(int iCnt = 0; iCnt < iSize; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }

        ArrayX aobj = new ArrayX();
        aobj.DivisibleBy5and3(Arr);

    }
}